src/utils/aeternity/wallet.js

```diff
@@ -8,128 +8,95 @@ import {
   MemoryAccount,
 } from '@aeternity/aepp-sdk'
 
-import { reactive, toRefs } from 'vue'
-import { COMPILER_URL } from './configs'
+import { reactive, toRefs, shallowReactive } from 'vue'
 
-export let sdk = null
-
-export const wallet = reactive({
-  activeWallet: null,
-  address: null,
+export const state = reactive({
+  walletInfo: null,
+  aeSdk: null,
   balance: null,
-  walletStatus: null,
-  isStatic: false,
-  networkId: null,
+  status: 'connecting',
+  networkId: process.env.VUE_APP_NETWORK_ID,
 })
 
 export const initWallet = async () => {
-  const { walletStatus } = toRefs(wallet)
-
-  const nodes = [{
-    name: process.env.VUE_APP_TYPE,
-    instance: new Node(process.env.VUE_APP_URL),
-  }]
-
-  walletStatus.value = 'connecting'
+  const { status, aeSdk } = toRefs(state)
 
   try {
-    const { VUE_APP_WALLET_SECRET_KEY, VUE_APP_WALLET_PUBLIC_KEY } = process.env
-    // connect to static Wallet
-    if (VUE_APP_WALLET_SECRET_KEY && VUE_APP_WALLET_PUBLIC_KEY) {
-      const account = new MemoryAccount({
-        keypair: { secretKey: VUE_APP_WALLET_SECRET_KEY, publicKey: VUE_APP_WALLET_PUBLIC_KEY },
-      })
+    const aeSdkOptions = {
+      nodes: [{
+        name: process.env.VUE_APP_NETWORK_ID,
+        instance: new Node(process.env.VUE_APP_NODE_URL),
+      }],
+      compilerUrl: process.env.COMPILER_URL,
+    }
 
-      const client = new AeSdk({
-        compilerUrl: COMPILER_URL,
-        nodes,
-      })
+    // connect to static Wallet
+    if (process.env.VUE_APP_WALLET_SECRET_KEY) {
+      const account = new MemoryAccount(process.env.VUE_APP_WALLET_SECRET_KEY)
 
-      await client.addAccount(account, {select: true})
-      sdk = client
+      // AeSdk instance can't be in deep reactive https://stackoverflow.com/a/69010240
+      aeSdk.value = shallowReactive(new AeSdk({
+        ...aeSdkOptions,
+        accounts: [account],
+      }))
 
-      walletStatus.value = 'connected'
-      await fetchWalletInfo(client)
+      await fetchAccountInfo()
     } else {
```

Comment:
I am not sure if this is really required. Examples should be short and precise. Memory account wallet example in real life is not feasible. A wallet for every AEPP doesn't make sense. I believe developers are looking for a Browser extension wallet connection from their AEPPs. If a Static wallet example is required then we can provide one more example repository by keeping both projects minimalistic.

Reply:
This is out of scope of this PR, I' would discuss it in a new issue. As I understand, `MemoryAccount` is used for development, to don't confirm connection to wallet each time 🙂 or running in e2e test. Old versions of cypress wasn't compatible with extensions.

Reply:
If it is for development then it's better to choose the right environment variable rather than introduce a new one
Intead `process.env.VUE_APP_WALLET_SECRET_KEY` of use `process.env.NODE_ENV`. Perhaps a few lines about this on README would be helpful.